source/reference/command/shardCollection.txt

```diff
@@ -194,6 +253,20 @@ avoid scalability and perfomance issues.
   - :ref:`sharding-shard-key-selection`
   - :ref:`sharding-shard-key`
 
+Shard Keys on Time Series Collections
+`````````````````````````````````````
+
+When sharding time series collections, you can only specify the
+``metaField``, ``timeField``, or both in the shard key. No other fields,
```

Comment:
We can also add `metaField (or sub-fields of a metaField)` to clarify that the field need not be just the `metaField`?